### ISSUE TITLE
HTTP Accept header only accepts singular value

### ISSUE THREAD

**Issue by quick-pawiromitchel** [sigp/lighthouse#4358]
## Description
The axios library passes multiple Accept Header values like so
```bash
curl -H 'Accept: application/json, text/plain, */*' -H 'User-Agent: axios/0.27.2' -H 'Connection: close' IP:PORT
```
This results in
```json
{
  "code": 400,
  "message": "BAD_REQUEST: Unsupported endpoint version: v2",
  "stacktraces": []
}
```

Replacing the Accept header with a single value like `application/json`, the call will work as intended.
A similar issue is found here https://github.com/sigp/lighthouse/issues/3114

## Version
`Lighthouse v4.1.0-693886b`

## Steps to resolve
By overriding the `Accept` header, the call will go through successfully
```js
const axios = require('axios');

axios.get('http://IP:PORT/eth/v2/beacon/blocks/finalized', {
    headers: {
        'Accept': '*/*'
    }
})
    .then(data => {
        console.log('Data:', data);
    })
    .catch(error => {
        console.error('There was a problem with the fetch operation:', error);
    });
```

**Comment by quick-pawiromitchel** [comment]
Related to https://github.com/sigp/lighthouse/issues/3114

**Comment by michaelsproul** [comment]
Should be fixed by https://github.com/sigp/lighthouse/pull/4216

I'll try to get that reviewed for v4.2.1/v4.3.0

**Comment by quick-pawiromitchel** [comment]
Awesome, thank you 💯

**Comment by jmcph4** [comment]
@quick-pawiromitchel This seems solved now that #4216 has landed in [4.3.0](https://github.com/sigp/lighthouse/releases/tag/v4.3.0).

For me locally:

```
$ curl -H 'Accept: application/json, text/plain, */*' -H 'User-Agent: axios/0.27.2' -H 'Connection: close' 0.0.0.0:8080
{"code":404,"message":"NOT_FOUND","stacktraces":[]}
```
